### ISSUE TITLE
Create the simplest sample that uses RNW in a Xaml islands

### ISSUE THREAD

**Issue by chrisglein** [microsoft/react-native-windows-samples#540]
Xaml Islands are a complicated topic, and we get lots and lots of questions about them ([and bugs that happen on them](https://github.com/microsoft/react-native-windows/issues?q=is%3Aopen+is%3Aissue+label%3A%22Area%3A+Islands%22)).

- There's a nice [article](https://techcommunity.microsoft.com/t5/windows-dev-appconsult/getting-started-with-react-native-for-windows/ba-p/912093) that walks through adding RNW to a WPF app using Xaml Islands. But that's a slice in time, currently 2 years old, and not accessible as a sample that people can quickly build and copy.
- There's an existing repo of [Xaml Islands Samples](https://github.com/microsoft/xaml-islands-samples), but that's not going to include RNW.
- We have [another issue](https://github.com/microsoft/react-native-windows-samples/issues/94) on this repo tracking the need for a write up on this topic.

I think what need is the simplest example app we can muster of a Xaml islands + RNW app. And we need it to live here on this repo alongside our other samples.

**Comment by asklar** [comment]
To me this is by design, per discussion in microsoft/react-native-windows-samples#267

**Comment by chrisglein** [comment]
> To me this is by design, per discussion in #267

By which you mean that TodosFeed should suffice? It isn't minimal, and has been locked in time to match the article it goes with.

At the very least, we don't point people in a clear direction if they're looking to use RNW with XAML Islands, and it comes up a lot. Whatever we thinking is exemplary here should also be discoverable.

**Comment by asklar** [comment]
no, what I'm saying is we probably don't want 3rd parties to do xaml islands without some white gloving. If you remember the conversation we had in 267, we had decided to not publicize xaml islands for RNW since there are lots of gotchas and we are not resourced to support it for arbitrary 3rd parties.
